android/app/src/main/java/io/mosip/registration_client/api_services/CommonApiImpl.java

```diff
@@ -0,0 +1,50 @@
+package io.mosip.registration_client.api_services;
+
+import androidx.annotation.NonNull;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+import io.mosip.registration.clientmanager.dto.registration.GenericValueDto;
+import io.mosip.registration.clientmanager.spi.MasterDataService;
+import io.mosip.registration_client.model.CommonApiPigeon;
+
+@Singleton
+public class CommonApiImpl implements CommonApiPigeon.CommonApi {
```

Comment:
Why is this class named as CommonApiImpl ? Can we name it as CommonDetailsApi. Let's not change the convention.

Reply:
Changed name to CommonDetailsApi